### PR TITLE
iSeqnorm pooling notebook. 

### PR DESCRIPTION
**Pull request by RodolfoSalido**
I made a new pooling notebook for the iSeqnorm pipeline. I added new visualizations and wrote 4 functions that could be merged into metapool.py:
calculate_iseqnorm_pooling_volumes()
linear_transform()
read_survival()
merge_read_counts() 

Unfortunately, I haven't written any tests for them. 

Let me know if you need anything from me.

### CODE REVIEW

**Review by charles-cowart — CHANGES_REQUESTED** [review]
Hi @RodolfoSalido ! I reviewed your new notebook manually since it was too big to review in GitHub. Everything looks good and ran correctly. I'm going to approve it as-is. I think @mmbryant23 may have suggestions such as breaking this up into smaller notebooks. Those diffs are likely to be smaller and should work better for reviewing.

I have one request - can you please change sklearn in setup.py to be 'scikit-learn' instead? That will prevent an error where scikit-learn isn't installed for new users.

File: metapool/metapool.py [hunk 1]
```diff
@@ -288,7 +288,8 @@ def read_plate_map_csv(f, sep='\t', qiita_oauth2_conf_fp=None):
 
 
 # method to read minipico output
-def read_pico_csv(f, sep='\t', plate_reader='Synergy_HT',
+def read_pico_csv(f, sep='\t', plate_reader='SpectraMax_i3x',
+                  min_conc = 0, max_conc = 150,
```
[the remark this answers is not in the document]
Comment (RodolfoSalido) [reply metapool/metapool.py]: addressed in [70c8568](https://github.com/biocore/metagenomics_pooling_notebook/pull/89/commits/70c85687f7ebc300c2fe8557ce1cc4514518e1fd)

**Comment by charles-cowart** [comment]
@RodolfoSalido The number of files to review jumped to 27. I'm not sure if this is because I inadvertently merged master into it in order to manually resolve a conflict. Looking into it.

**Comment by charles-cowart** [comment]
Hi @RodolfoSalido in test_metapool.py, there are several lines calling the join() function instead of os.path.join() or defining 'from os.path import join' at the top of the file. Can you please change all occurrences to use either join() or os.path.join()? Please note that there are also a few lines that are calling string.join() which is a different function entirely so you'll want to be careful with search and replace. This should get most of the tests to pass.

**Comment by charles-cowart** [comment]
Hi @RodolfoSalido it looks like there are four tests failing and one test has an error. It looks like at least some of them such as test_estimate_read_depth(), the observed results from calculate_iseqnorm_pooling_volumes() differ from the expected values of the TSV file by a single row or column. Can you please confirm that the following tests work as intended?
test_estimate_read_depth
test_find_threshold_autopool
test_merge_read_counts
test_read_pico_csv
test_read_plate_map_csv_validate_qiita_sample_names

You can review the error messages here: https://github.com/biocore/metagenomics_pooling_notebook/actions/runs/4611263674/jobs/8150742325?pr=89

**Comment by charles-cowart** [comment]
@RodolfoSalido it looks like we are down to one error and two failures. I'll pull the latest and see if I can't resolve these. If I have any questions I'll ping you. Thanks!

**Comment by charles-cowart** [comment]
@RodolfoSalido It looks like earlier today you pushed a commit with only a few file changes, followed by a second commit later that added most of the 27 new files that we're seeing. Can you confirm that this is correct? For the record I am usually in favor of adding as many sample input/output files as we need.

**Comment by charles-cowart** [comment]
@RodolfoSalido I pushed a fix for a single test to confirm that I could push to your repo. I'll be pushing fixes for the remaining tests as soon as possible.

**Comment by charles-cowart** [comment]
> Yes, should be correct. The commit with the 27ish file changes included the unit_tests and a bunch of input and output files.
> […](#)

Great, thanks Rodolfo!

**Comment by charles-cowart** [comment]
@RodolfoSalido All tests are passing! I will approve and merge.

**Review by charles-cowart — APPROVED** [review]

**Pull request merged**